reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Does this create an additional stack each iteration? If so, then this will start causing issues related to stack limits for long running environments. It might be better to just use the native ansible modules for scale-up until ASGs are in use.